### PR TITLE
updated artifacts for ldap extended tests

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Follow-up to #5047 

cc/ @deads2k

### CODE REVIEW

File: test/extended/authentication.sh [hunk 1]
```diff
@@ -77,16 +77,79 @@ echo "[INFO] Running extended tests"
...
-oc delete groups --all
-
-cp test/extended/authentication/sync-schema2-01.yaml ${BASETMPDIR}
-os::util::sed "s/LDAP_SERVICE_IP/${LDAP_SERVICE_IP}/g" ${BASETMPDIR}/sync-schema2-01.yaml
-openshift ex sync-groups --sync-config=${BASETMPDIR}/sync-schema2-01.yaml --confirm
-
-
-# Run the tests
-#LDAP_IP=${LDAP_SERVICE_IP} TMPDIR=${BASETMPDIR} ginkgo -progress -stream -v -focus="authentication: OpenLDAP" ${OS_OUTPUT_BINPATH}/extended.test
+schema=('rfc2307' 'ad')
```
Comment (deads2k) [inline test/extended/authentication.sh]: I thought I already had this change in your previous pull.

Comment (stevekuznetsov) [inline test/extended/authentication.sh]: This is dependent on the previous pull and the real diff will become obvious once it goes in and the diff is between this and that.

**Comment by danmcp** [comment]
@stevekuznetsov Needs rebase

**Comment by deads2k** [comment]
let me know when this is ready.

**Comment by stevekuznetsov** [comment]
@deads2k this is ready

File: test/extended/authentication.sh [hunk 2]
```diff
@@ -101,7 +101,7 @@ function compare_and_cleanup() {
 	validation_file=$1
 	actual_file=actual-${validation_file}.yaml
 	rm -f ${WORKINGDIR}/${actual_file} 
-	oc get groups --no-headers -o name | sort | xargs -I{} oc export {} -o yaml >> ${WORKINGDIR}/${actual_file}
+	oc get groups --no-headers | awk '{print $1}' | sort | xargs -I{} oc export group {} -o yaml >> ${WORKINGDIR}/${actual_file}
```
Comment (deads2k) [inline test/extended/authentication.sh]: no, use `-o name`

**Comment by deads2k** [comment]
[merge] to get this closer.  fix it up again tomorrow.

File: test/extended/authentication.sh (hunk 2 above)
Comment (stevekuznetsov) [inline test/extended/authentication.sh]: It's currently broken...

Comment (stevekuznetsov) [inline test/extended/authentication.sh]: Using `-o name` is broken as documented in #5255

**Pull request merged**